utils/defaults/defaults.go

```diff
@@ -116,6 +118,13 @@ func GetExperimentProgressDeadlineSecondsOrDefault(e *v1alpha1.Experiment) int32
 	return DefaultProgressDeadlineSeconds
 }
 
+func GetAbortExceedProgressDeadlineOrDefault(rollout *v1alpha1.Rollout) bool {
+	if rollout.Spec.AbortExceedProgressDeadline != nil {
+		return *rollout.Spec.AbortExceedProgressDeadline
+	}
+	return DefaultAbortExceedProgressDeadline
+}
+
```

Comment:
This will be unnecessary if we make progressDeadlineAbort a bool and not a bool pointer.

Reply:
Removed